src/services/getPreparedTodos.ts

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../types/Todo';
+
+export const getPreparedTodos = (
+  todos: Todo[],
+  filterField: string,
+  query: string,
+) => {
+  let visibleTodos = [...todos];
+
+  if (query) {
+    visibleTodos = visibleTodos.filter(todo =>
+      todo.title.toLowerCase().includes(query.toLowerCase().trim()),
+    );
+  }
+
+  if (filterField) {
+    switch (filterField) {
+      case 'active':
+        visibleTodos = visibleTodos.filter(todo => todo.completed === false);
```

Comment:
```suggestion
        visibleTodos = visibleTodos.filter(todo => !todo.completed);
```
Don't compare with false/true, the todo.completed value is already a boolean